formal-resume/resume_english.tex

```diff
@@ -75,14 +75,17 @@ \section{Honors}
 \section{Education}
 
 \datedline{\textbf{Changshu Institute of Technology}, China}{Sep 2016 -- Jul 2020}
-  Bachelor of Engineering, Automotive Service Engineering. I also have passed the CET-6 at this period.
+\begin{itemize}
+  \item Bachelors of Engineering, Automotive Service Engineering
+  \item Completed the College English Test at the CET6 level
```

Comment:
You may want to consider a section "Language" section:

Languages
- Mandarin, Fluent
- English, Professional Level (CET6)
- ...

Reply:
I think I will consider creating the Language section to mention that instead of mentioned in the Education section.

Kenji Mouri

Reply:
Finally, I added languages to the skills section in https://github.com/MouriNaruto/MouriNaruto/commit/11cc4094ac9718661b2e850f581745b9f73c0096